nautobot/extras/migrations/0063_rename_model_fields.py

```diff
@@ -0,0 +1,54 @@
+# Generated by Django 3.2.16 on 2023-02-08 23:19
```

Comment:
Does this need to be two separate migrations files, or could they be merged?

Reply:
If I delete both migration files and rerun `makemigration` Django thinks I'm creating new field for `custom_field` and `secrets_group`